src/app/examples/card-example/card-example.component.html

```diff
@@ -6,5 +6,16 @@
     Lorem, ipsum dolor sit amet consectetur adipisicing elit. Mollitia facere molestias recusandae necessitatibus ab veniam repellendus doloremque culpa quam libero, est quo accusamus cumque, in quia itaque cupiditate ratione repellat!
   </div>
 
+  <kirby-card-footer></kirby-card-footer>
+</kirby-card>
+
+<kirby-card>
+  <kirby-card-header title="Padding" subtitle="Card with padding"></kirby-card-header>  
```

Comment:
I would like the example to be reverted as well, so we don't have a case of "padding" which is default but a case special case showing you can remove the padding

Reply:
Ok, good point.